js/ooyala_ssai.js

```diff
@@ -1358,36 +1370,27 @@ OO.Ads.manager(function(_, $)
      * @param {object} urlObject An object with the tracking event names and their
      * associated URL array.
      */
-    var _pingTrackingUrls = _.bind(function(urlObject)
-    {
-      for (var trackingName in urlObject)
-      {
-        if (urlObject.hasOwnProperty(trackingName))
-        {
-          try
-          {
-            var urls = urlObject[trackingName];
-            if (urls)
-            {
-              if (bustTheCache)
-              {
-                urls = _cacheBuster(urls);
-              }
-              OO.pixelPings(urls);
-              OO.log("Ooyala SSAI: \"" + trackingName + "\" tracking URLs pinged");
-            }
-            else
-            {
-              OO.log("Ooyala SSAI: No \"" + trackingName + "\" tracking URLs provided to ping");
-            }
+    var _pingTrackingUrls = bind(function(urlObject) {
```

Comment:
Would you like to change it (from var to const/let)?

Reply:
I think it's better to do this with adding eslint